app/views/static_pages/home/_how_it_works.html.erb

```diff
@@ -1,34 +1,31 @@
+ <% tiles = [
```

Comment:
I can do that.  It almost seems _tooooo_ abstracted, but doesn't hurt my feelings.  It certainly makes teh component cleaner